src/trace/implementations/ord.rs

```diff
@@ -616,6 +608,8 @@ where
 
 		if let Some(frontier) = frontier.as_ref() {
 			OrdKeyBatch::advance_builder_from(&mut self.result, frontier, initial_key_pos);
+			let new_since = antichain_join(self.description.since(), &frontier[..]);
+			self.description = Description::new(self.description.lower(), self.description.upper(), new_since.elements());
```

Comment:
Possibly also redundant, this time with the lines at 545, but for the same reason as further above.